content/blog/configuring-sso-for-hpe-greenlake-central-private-cloud-enterprise-and-hpe-greenlake-glcp-using-okta.md

```diff
@@ -0,0 +1,162 @@
+---
+title: How to implement a single sign-on solution to authenticate users onto the
+  HPE GreenLake edge-to-cloud platform
+date: 2023-11-29T11:34:12.167Z
+featuredBlog: false
+externalLink: ""
+author: Prabhu Murthy - Cloud Operations
+authorimage: /img/Avatar1.svg
+thumbnailimage: /img/hpe-greenlake-sso-page.jpg
+disable: false
+tags:
+  - HPE GreenLake Private Cloud Enterprise application
+  - HPE GreenLake edge-to-cloud platform
+  - sso
+  - okta
+  - Active Directory
+  - hpe-greenlake
+---
+Enterprises looking to use HPE GreenLake for Private Cloud Enterprise can benefit from the use of SSO, as it has been integrated onto the HPE GreenLake edge-to-cloud platform, which supports single sign-on
+
+In this blog post, I will walk you through the process of configuring Okta Active Directory (AD) to authenticate users into the HPE GreenLake for Private Cloud Enterprise application on the HPE GreenLake platform using SAML Identity Provider (IdP) for single sign-on.
+
+### Before starting
+
+Please review the [HPE GreenLake](https://support.hpe.com/hpesc/public/docDisplay?docId=a00120892en_us&page=GUID-D7192971-EF71-4304-B51E-548E7954E644.html) User Guide to understand how the SAML framework works in the context of Common Cloud Services for the HPE GreenLake edge-to-cloud platform for the HPE GreenLake Private Cloud Enterprise application .
+
+### Configure SSO/SAML applications in Okta
+
+   To configure application metadata in Okta, complete the following steps:
+
+* Step 1: Create an Okta SAML application
+* Step 2: Configure Sign On settings
+* Step 3: Export the SAML 2.0 IdP metadata
+* Step 4: Configure the SAML connection in the HPE GreenLake edge-to-cloud platform
+
+**Step 1: Create an Okta SAML application**
+
+1. Log into the Okta administration console.
+2. Click **Applications > Create new app integration.** The Create a new app integration window opens.
+3. Select SAML 2.0 and click **Next**.
+
+![](/img/ws-image0.png)
+
+Provide a name for the SAML application which gets connected to the HPE GreenLake edge-to-cloud platform
+
+![](/img/saml_app-okta.jpg)
+
+**Step 2: How to configure single sign-on settings**
+
+1. Enter the SAML information.
+
+   Under General:
+
+    **Single Sign on URL:** https://sso.common.cloud.hpe.com/sp/ACS.saml2
+
+    **Audience URI (SP Entity ID):** https://sso.common.cloud.hpe.com
+
+    **Name ID format EmailAddress**
+
+    **Application username Email**
+
+    **NameID = user.email**
+
+    **gl_first_name = user.FirstName**
```

Comment:
For variable with "underscore" make sure to escape special character such as the underscore with a backslach:

for example: gl\_first\_name , gl\_last\_name, hpe\_ccs\_attribute